scripts/seed/itemData.js

```diff
@@ -0,0 +1,49 @@
+const items = [
+    {
+        title: "Washing Machine (Small)",
+        postType: "Request",
+        description:
+            "Low-income family of 3 living in a 1-room rental flat is in need of a small washing machine (due to the limitation in space) to ease their daily stress over household chores. Father is currently overwhelmed with the caregiving and financial stressors which have arisen from mother (on LTVP)'s recent foot surgery.",
+        category: "Appliances",
+        delivery: "Included",
+        status: "Open",
+        tags: "washing machine, appliances",
+        postedBy: "5fb8b003e1e86a126a37520d",
```

Comment:
not sure why but the postedBy entries in the db are not being converted to ObjectId format...

Reply:
Rectified by re-seeding itemData